charts/argo-services/scripts/check-for-promotion.sh

```diff
@@ -2,8 +2,8 @@
 set -euo pipefail
 
 # Fetch the YAML configuration
-CONFIG_URL="https://raw.githubusercontent.com/alphagov/govuk-helm-charts/main/charts/app-config/image-tags/${ENVIRONMENT}/${REPO_NAME}"
-CONFIG_CONTENT=$(curl -Ls "${CONFIG_URL}")
+CONFIG_URL="https://api.github.com/repos/alphagov/govuk-helm-charts/contents/charts/app-config/image-tags/${ENVIRONMENT}/${REPO_NAME}?ref=main"
```

Comment:
I'm not sure this is entirely immune to caching either:

```
$ curl -v 'https://api.github.com/repos/alphagov/govuk-helm-charts/contents/charts/app-config/image-tags/production/frontend?ref=main' 2>&1 | grep cache
< cache-control: public, max-age=60, s-maxage=60
```

I wonder if there's some way to avoid this race condition entirely 🤔